### PR TITLE
adicionado input reader ao main

### PR DESCRIPTION
**Pull request by arimateia98**

### CODE REVIEW

File: src/Main.ts [hunk 1]
```diff
@@ -1,34 +1,41 @@
 import {ControllerSer} from './ControllerSer';
-const readline = require('readline');
+const readline = require('readline-sync');
```
Comment (FMota0) [inline src/Main.ts:2]: This wasn't added in package.json 
`npm i --save x` to add x as a dependencie
`npm i --save-dev x` to add x as a dev-dependencie

File: src/Main.ts [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
-        console.log("c - create \n u - update \n d - delete \n 1 - read one \n a - read all \n f - fim \n");
-        var opcao:string ;
-        switch(opcao){
+        var controller:ControllerSer = new ControllerSer();''
+        var opcao:string = readline.question("c - create ser \n u - update ser \n d - delete ser \n a - read all ser \n 1 - read one ser \n e - end\n ");
+        while(opcao != "e"){
+            switch(opcao){
                 case "c":
-                    console.log("c")
+                    var id:number = readline.question("enter the ip");
```
Comment (FMota0) [inline src/Main.ts]: Add a space after the question or print it and end the line, it's not good to have something like `enter the ip1`

File: src/Main.ts [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
-                case "f":
-                    controller.saveAll();
-                    break;
-                    
+                
+            }
+            opcao = readline.question("c - create ser \n u - update ser \n d - delete ser \n a - read all ser \n 1 - read one ser \n e - end\n ");
         }
-        console.log("c - create \n u - update \n d - delete \n 1 - read one \n a - read all \n f - fim \n");
+        controller.saveAll();
```
Comment (FMota0) [inline src/Main.ts:38]: Is this really working? I couldn't see it working, after I ended a execution in the next one the list was empty.

File: src/Main.ts (hunk 1 above)
Comment (arimateia98) [reply src/Main.ts:2]: Done

File: src/Main.ts (hunk 2 above)
Comment (arimateia98) [reply src/Main.ts]: Done

File: src/Main.ts (hunk 3 above)
Comment (arimateia98) [reply src/Main.ts:38]: At least here it's working.

**Comment by FMota0** [comment]
https://github.com/arimateia98/CrudTS/blob/e9311db7a15d5318f9252c4788e44dfa4fe08b1f/src/ControllerSer.ts#L8 I don't think require should be used in this way, I think the best way is to create the file depending if exists or not and then read with `fs.readFileSync`

**Comment by FMota0** [comment]
https://github.com/arimateia98/CrudTS/blob/e9311db7a15d5318f9252c4788e44dfa4fe08b1f/src/ControllerSer.ts#L49 I think the path should be `./array.json`, because it doesn't work if you are not inside the src directory and this path should be in a constant

**Comment by FMota0** [comment]
https://github.com/arimateia98/CrudTS/blob/e9311db7a15d5318f9252c4788e44dfa4fe08b1f/src/ControllerSer.ts#L48 In some moments var is being used in others let, const , should have a pattern

**Comment by FMota0** [comment]
I think it can improve a lot in basic things, adding a README would be awesome and having some way to run it like `npm start`

**Pull request closed without merging**